app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        special_skills = ["JavaScript", "HTML", "CSS"]
+        for skill in special_skills:
```

Comment:
Here u can do like this self.skills += ["JavaScript", "HTML", "CSS"] or use `extend` to write less code
check in all places